packages/core/src/Modules/ComplaintsPolicy/Components/complaints-policy-content.jsx

```diff
@@ -187,32 +323,36 @@ const Content = () => {
         },
     ];
 
-    const modal_content = [...policy_content, ...complaints_procedure].map((row, index) => (
-        <div key={index} className='complaints-policy__section'>
-            <div className='complaints-policy__section-title'>{row.title}</div>
-            {row.list && (
-                <div className='complaints-policy__list'>
-                    {row.list.map((item, i) => (
-                        <div key={i} className='complaints-policy__list-item'>
-                            {item}
-                        </div>
-                    ))}
-                </div>
-            )}
-            {row.content && (
-                <div className='complaints-policy__section-content'>
-                    {row.content.text && row.content.text}
-                    {/* eslint-disable-next-line react/display-name */}
-                    {row.content.subcontent?.map((item, i) => (
-                        <div key={i} className='complaints-policy__subsection'>
-                            <div className='complaints-policy__subsection-title'>{item.title}</div>
-                            <div className='complaints-policy__subsection-content'>{item.text}</div>
-                        </div>
-                    ))}
-                </div>
-            )}
-        </div>
-    ));
+    const can_show_complaints_procedure = /^(svg|labuan|vanuatu|maltainvest)$/.test(landing_company_shortcode);
+
+    const modal_content = [...policy_content, ...(can_show_complaints_procedure ? complaints_procedure : [])].map(
+        (row, index) => (
+            <div key={index} className='complaints-policy__section'>
+                <div className='complaints-policy__section-title'>{row.title}</div>
+                {row.list && (
+                    <div className='complaints-policy__list'>
+                        {row.list.map((item, i) => (
+                            <div key={i} className='complaints-policy__list-item'>
+                                {item}
+                            </div>
+                        ))}
+                    </div>
+                )}
+                {row.content && (
+                    <div className='complaints-policy__section-content'>
+                        {row.content.text && row.content.text}
```

Comment:
`{row.content.text}` Should be fine here since `undefined` won't be rendered.

Reply:
I've removed it. Thanks! 🙇